html_css_dom/axios+async-await/README.md

```diff
@@ -207,7 +217,6 @@ Copy this code to your test.js to see it in action:
   fireRequest();
 
 ```
-
 # 5. Building an app with await async
 
 Let's return to the application that we built out with `Fetch`.
```

Comment:
Is this meant to be the same code from the Promises/Fetch lesson? I think the code was changed there but not here. We don't have to change it here as well but should clarify that this is not exactly the same application.

Reply:
Yes, it's meant to be the same code. I removed it from Promises / Fetch b/c I didn't believe that it really helped demonstrate fetch. I felt that all the code there in a README would add to a lot of confusion. I actually wanted to remove it from here as well b/c I don't think it's really needed. But I wasn't sure if I should be making that decision / how much of the lesson I should be rewriting. 
I personally don't find walls of code helpful in reading environments. I feel uncertain about the purpose of the curriculum and how fellows and staff should be interacting with it and using it. Is it meant to read like a book for fellows? Should it primarily be prereading for fellows or is this a main place to learn? Is it  a lesson with interactive examples or are they reading it on the subway? Do instructors use these examples or build their own? Because of these questions, I sometimes feel limited in what I can do to change the curriculum. Also the questions of should I be making massive changes before we hire someone that may have a different vision?

Reply:
Great questions! I say go ahead and kill it.
Don't worry about the person who we haven't hired and who doesn't work here yet. There is GitHub history if that person wants to restore anything.
I think all of the instructors would agree that the wall of code is not helpful here. (@myrasmith just mentioned it to me).

Reply:
Do you still want this to API keys here? The current example doesn't show how to actually import the key while it's hidden. I'm looking into good solutions to this now without introducing different tech. But does this belong in the curriculum at this time? Also working on finding a different simple example, the current API for Dark Sky no longer allows people to sign up.

Reply:
It sounds like we can do these things:
1. Remove anything about the Dark Sky API
2. After we do 1), there won't be any more need to create and store API keys, so we can get rid of the information about hiding API keys.
Does that sound right?

We will want to teach them how to handle API keys eventually but it does not need to be here.

Reply:
I added it to a things to Note. And used the example from the fetch lesson updated. Noting the differences.